src/main/java/org/havenapp/main/Utils.java

```diff
@@ -11,9 +11,9 @@
  * Class containing util functions which will be used multiple times throughout the app.
  */
 
-class Utils {
+public class Utils {
 
-    private static final String DATE_TIME_PATTERN = "yyyy-MM-dd_HH:mm:ss";
+    public static final String DATE_TIME_PATTERN = "yyyy-MM-dd_HH-mm-ss.SSS";
```

Comment:
Do we need a different character for seconds and milliseconds?

Reply:
Not sure what you mean by different characters?  Do you mean the period separating them?  Since it deviates from the ISO8601 standard, I wanted to make it clear that these were a fraction of a second.

So a sample path would look like:

`/sdcard/phoneypot/2018-09-26_21-17-41.059/`

and a file would look like:

`2018-09-26_21-06-52.501.m4a`
`2018-09-26_21-09-00.918.mp4`
etc.

These should be listable, sortable, and draggable to any OS.  And all the files should be there, even if two are saved within the same (unique) second.

Reply:
That said-- please try it to make sure files are saving properly in the correct place.

Reply:
Yes, I was unaware of this period/dot convention. Thanks for this clarification. :)

Reply:
It's not exactly a convention-- it's a fraction of a second.  Originally I used a dash, but it got a little confusing because then it's a lot of numbers with dashes in 'em...  Anyway, please give it a shot and let me know.  If you want to change the formatting, it's all in a single place and can be changed easily-- assuming it all works.  Maybe @deviantollam can chime in..

Reply:
i've always seen it done with a period if someone is specifying milliseconds

the ISO 8601 standard specifies a period for this usage, too.

YYYY-MM-DD_HH-MM-SS.ssssZ